include/materials/norsand.tcc

```diff
@@ -62,6 +93,8 @@ mpm::NorSand<Tdim>::NorSand(unsigned id, const Json& material_properties)
     Mtc_ = (6 * sin_friction_cs) / (3 - sin_friction_cs);
     Mte_ = (6 * sin_friction_cs) / (3 + sin_friction_cs);
 
+    // TODO: Need to find a way how to compute Lambda for Bolton's CSL eq
+    // Lambda volumetric
     chi_image_ = chi_ / (1. - ((chi_ * lambda_) / Mtc_));
```

Comment:
@jgiven100 any idea on how to compute lambda for this? Do you think it's good to add the bolton's CSL, or delete it for now?

Reply:
Due to the trouble with `e_min` and `e_max`, Jefferies seems pretty against Bolton's CSL. I am likely to follow Jefferies' advice.

But someone else might want to use Bolton's CSL (e.g., you a few months ago), so why not leave it in? As is, even if someone is using Bolton's CSL, they are expected to give `lambda`; this is acceptable for me. Considering most user will likely be using a look-up table to determine `e_min` and `e_max`, there are many look-up tables with average values of `lambda`.

Reply:
Okay. I think that's possible! I will remove the TODO